src/TheThingsNetwork.cpp

```diff
@@ -44,11 +44,12 @@ bool TheThingsNetwork::sendCommand(String cmd) {
 }
 
 bool TheThingsNetwork::sendCommand(String cmd, String value) {
-  int l = value.length();
-  byte buf[l];
-  value.getBytes(buf, l);
+  String str = "";
```

Comment:
Well, as it is before this PR it uses the still existing `TheThingsNetwork::sendCommand(String cmd, const byte *buf, int length)`, not this `TheThingsNetwork::sendCommand(String cmd, String value)` right?